cryptofeed/rest/gdax.py

```diff
@@ -0,0 +1,256 @@
+import time, json, hashlib, hmac, calendar, requests, base64
+from time import sleep
+from datetime import datetime as dt
+
+import pandas as pd
+
+from cryptofeed.rest.api import API
+from cryptofeed.feeds import GDAX
+from cryptofeed.log import get_logger
+from cryptofeed.standards import pair_std_to_exchange, pair_exchange_to_std
+
+REQUEST_LIMIT = 10
+LOG = get_logger('rest', 'rest.log')
+
+# API Docs https://docs.gdax.com/
+class Gdax(API):
+    ID = GDAX
+    # api = "https://api.gdax.com"
+    api = "https://api-public.sandbox.gdax.com"
```

Comment:
maybe have a way to select sandbox vs real api? Default should be real, but maybe change API's `__init__` to allow a sandbox=True kwarg